docs/source/tools/navigator/index.rst

```diff
@@ -247,6 +247,24 @@ To advance time:
   :width: 25%
   :align: center
 
+Authenticating Navigator
+************************
+
+If you are running Navigator against a Ledger API server that requires authentication, you must provide the access token when you start the Navigator server.
+
+The access token retrieval depends on the specific DAML setup you are working with: please refer to the ledger operator to learn how.
```

Comment:
Maybe also mention the exact content of the file, like whether it will have `Bearer ` prepended?

Reply:
That is specific to the setup, pointing the reader to the ledger operator is the only thing we can do reliably.